platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/problems/ConfigurationCacheProblemsSummary.kt

```diff
@@ -54,29 +53,31 @@ class ConfigurationCacheProblemsSummary(
     var overflowed: Boolean = false
 
     private
-    var problemCount: Int = 0
+    var totalProblemCount: Int = 0
 
     private
-    var failureCount: Int = 0
+    var deferredProblemCount: Int = 0
 
     private
-    var suppressedCount: Int = 0
+    var suppressedProblemCount: Int = 0
```

Comment:
💅 Here total is a total, right?  Total = deferred + suppressed?  This would be my expectation throughout this PR.